onyo/cli/tsv_to_yaml.py

```diff
@@ -22,12 +22,13 @@
 epilog_tsv_to_yaml = r"""
 .. rubric:: Examples
 
+Convert a table to a YAML file:
```

Comment:
I bump on the word "convert", since it doesn't convert the file.

`tsv_to_yaml` prints a TSV file as YAML.

And what it produces is YAML text, not "a YAML file".